R/Execution.R

```diff
@@ -41,10 +48,14 @@
 execute <- function(analysisSpecifications,
                     executionSettings,
                     executionScriptFolder = NULL,
+                    keyringName = NULL,
+                    keyringPassword = NULL,
```

Comment:
I plan to remove this parameter in favor of using Sys.getenv("STRATEGUS_KEYRING_PASSWORD") so that this information is not passed around in any scripts.